kubernetes/assets/recommended_monitors/monitor_containers_terminated.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "[kubernetes] Monitor Kubernetes Pods Terminated",
+	"type": "query alert",
+	"query": "change(sum(last_5m),last_5m):avg:kubernetes.containers.state.terminated{*} by {kubernetes_cluster} >= 1",
```

Comment:
Where is that metric coming from? It's not in the metadata.csv for this integration. https://github.com/DataDog/integrations-core/blob/master/kubernetes/metadata.csv

Also using the `avg` space aggregator might not do what you want here.